test/parallel/test-event-emitter-errors.js

```diff
@@ -6,5 +6,5 @@ var assert = require('assert');
 var EE = new EventEmitter();
 
 assert.throws(function() {
-  EE.emit('error', 'Accepts a string');
-}, /Accepts a string/);
+  EE.emit('error', {message: 'Error!'});
+}, /Unhandled "error" event/);
```

Comment:
Can you leave the existing code here. I'm fine with adding this code too though.